README.md

```diff
@@ -31,6 +32,8 @@ SERVICE_DOMAIN=localhost
 
 You can find the `API_BASE_URL` in [Concourse](https://cd.gds-reliability.engineering/teams/verify/pipelines/di-authentication-deployment) under the outputs within the deploy-oidc-api-{environment} job, where {environment} is the name of the environment you want to connect to.
 
+`TEST_CLIENT_ID` must be a client configured in the client registry of the environment you are connecting to.  Look in the `environment-client-registry` DynamoDb table and retrieve the value of the `ClientID` field for the relevant client.
```

Comment:
This doesn't help explain which client I should be choosing. I'm guessing I want one for which TestClient = 1?

Reply:
Not necessarily, that's a different thing to do with suppressing notifications for synthetic users, best choose the one configured to work with the stub

Reply:
Also while you're in the client registry you need to ensure your configured redirect_uri matches what's in the client registry